### DISCUSSION TITLE
Change HighlightAnnot filled color?

### DISCUSSION THREAD

**Discussion by jiezhoujenny** [pymupdf/PyMuPDF#853] [Q&A] [answered]
Hi 
I am using following code to change highlightAnnt filled color, it works, yet throw the error "mupdf: Highlight annotations have no IC property" and "warning: annot type has no fill color", can you please advise what can change the highlight annotation color without these warnings? Thank you!
`r = fitz.Rect(rec_ann) 
ann_page=doc2[ann_rec_lst_new[i][1]]
annot = ann_page.addHighlightAnnot(r)
annot.setColors(stroke=blue, fill=blue)
annot.update()`

**Comment by JorjMcKie** [comment, marked as answer]
Even though highlight annots fill an area with color, the normal color parameter `/C` is used for this - just like for other text makrer annots. The `stroke` parameter must be used.

**Reply by jiezhoujenny** [reply]
Many thanks for your reply! removing parameter fill and using annot.setColors(stroke=blue)  annot.update() could change the color without warnings.

**Reply by JorjMcKie** [reply]
Hm, you are right, learned something: HTML output ignores any drawings.
Thre is still hope: you could make a little intermediate unicolor image and insert this underneath to-be-highlighted text:
```python
pix = fitz.Pixmap(fitz.csRGB, (0, 0, 500, 500), 0)  # pixmap that is large enough
pix.setRect(pix.irect, (255,255,0))  # fill pixmap with yellow
# now for every rectangle you want to "highlight", do this:
page.insertImage(rect, pixmap=pix, keep_proportions=False,  # adjust image to rect dimension
    overlay=False)  # put underneath
# then output html of the page
```
HTML output **_should_** come closer to what you want ...

**Reply by jiezhoujenny** [reply]
Awesome!  Thank you!
